repose-aggregator/extensions/simple-rbac/src/main/resources/META-INF/schema/config/simple-rbac.xsd

```diff
@@ -0,0 +1,115 @@
+                        This is the name of the header where that roles are passed in.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="resources-file-name" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        This is the name of a file containing a list of resources, HTTP methods, and roles allowed to access them.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="enable-masking-403s" type="xs:boolean" use="optional" default="false">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Instead of returning a FORBIDDEN (403), return a NOT FOUND (404) or METHOD NOT ALLOWED (405) as appropriate.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="dot-output" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        dot output file for this validator
+                        DOT is a plain text graph description language.
+                        It is a simple way of describing graphs that both humans and computer programs can use.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="wadl-output" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        wadl output file for this validator
+                        WADL is the Web Application Description Language.
+                        It is a way of describing the API of a Web Application that both humans and computer programs can use.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <!-- http://stackoverflow.com/a/16674043/423218 -->
+        <xs:assert vc:minVersion="1.1" test="count((@resources-file-name, ./rbac:resources)) = 1"
+                   xerces:message="Exactly one of the resources attribute or resources element can be specified, but not both."
+                   saxon:message="Exactly one of the resources attribute or resources element can be specified, but not both."/>
+    </xs:complexType>
+
+    <xs:complexType name="DelegatingType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Whether or not you would like this filter to populate the delegation headers. Inclusion means you do.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="quality" type="QualityType" use="optional" default="0.1">
```

Comment:
this seems like a low default. it should be the same value as the validation filter, because it's basically fulfilling the same  role